contracts/strategies/examples/quadratic-voting/QVSimple.sol

```diff
@@ -198,15 +199,15 @@ contract QVSimple is BaseStrategy, RecipientsExtension, AllocatorsAllowlistExten
     /// @param _amount The amount to withdraw
     /// @param _recipient The address to withdraw to
     function _beforeWithdraw(address _token, uint256 _amount, address _recipient) internal virtual override {
-        if (distributionStarted) {
+        if (totalPayoutAmount != 0) {
```

Comment:
Hmm thinking a bit more about this, after distribution happened pool admin's wont be able to use withdraw ever since it will always revert hete

Reply:
@ilpepepig pointed out that it's safe to remove that hook implementation now, since now `withdraw` is handling the `poolAmount`